src/ray/gcs/client.h

```diff
@@ -98,6 +100,7 @@ class RAY_EXPORT AsyncGcsClient {
   std::unique_ptr<RedisAsioClient> asio_async_auxiliary_client_;
   std::unique_ptr<RedisAsioClient> asio_subscribe_auxiliary_client_;
   CommandType command_type_;
+
```

Comment:
You seem to have added a bunch of new lines like this. I'm ok with it either way assuming it passes linting, but is there a reason for this?